README.adoc

```diff
@@ -0,0 +1,183 @@
+image:https://maven-badges.herokuapp.com/maven-central/org.jboss.aerogear.unifiedpush/unifiedpush-parent/badge.svg[Maven Central (Maven Central), link=https://maven-badges.herokuapp.com/maven-central/org.jboss.aerogear.unifiedpush/unifiedpush-parent]
+endif::[]
+
+:toc:
+toc::[]
+
+== Overview
+
+The _AeroGear UnifiedPush Server_ is a server that allows sending push notifications to different (mobile) platforms and has support for:
+* https://developer.apple.com/library/content/documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/APNSOverview.html#//apple_ref/doc/uid/TP40008194-CH8-SW1[Apple’s APNs (HTTP/2)]
+* https://firebase.google.com/docs/cloud-messaging/[Firebase Cloud Messaging]
+
+image::https://raw.githubusercontent.com/aerogear/aerogear-unifiedpush-server/master/ups-ui-screenshot.pnghttps://raw.githubusercontent.com/aerogear/aerogear-unifiedpush-server/master/ups-ui-screenshot.png[Create Server,align="center"]
+
+== Project Info
+
+|===
+| License:        | Apache License, Version 2.0
+| Build:          | https://maven.apache.org[Maven]
+| Documentation:  | https://docs.aerogear.org
+| Issue tracker:  | https://issues.jboss.org/browse/AEROGEAR
+| Mailing lists:  | https://groups.google.com/forum/#!forum/aerogear[aerogear-dev]
+|===
+
+== Prerequisites
+
+|===
+| https://maven.apache.org/install.html[Maven]
+| https://docs.docker.com/install/[Docker]
+|===
+
+== Getting started
+
+=== Cloning the repository
+
+By the following commands you will create a local directory and clone this project.
+
+[source,shell]
+----
+git clone https://github.com/aerogear/aerogear-unifiedpush-server.git
+cd aerogear-unifiedpush-server
+----
+
+=== Running locally
+
+==== Using Docker (Standalone)
+
+By the following commands you will create a local directory and clone this project.
+
+[source,shell]
+----
+mvn clean install -DskipTests
+docker run -p 9999:8080 -it aerogear/unifiedpush-configurable-container:2.2.2-SNAPSHOT 
+----
+
+To access this server go to `http://localhost:9999/`.
+
+NOTE: The standalone is going to use H2 as database, AMQ provides for the Wildfly using to run the UPS and it will not use the https://www.keycloak.org/[Keycloak].
+
+==== Using Docker-Compose
+
+For your convenience, we do have an easy way of launch different configurations of the UPS, using our link:./docker-compose[Docker compose files]
+
+By the following command you can startup this project locally using Docker-Compose.
+
+[source,shell]
+----
+docker-compose -f docker-compose-v2.1.yaml up
+----
+
+IMPORTANT: There is a bug on docker-compose for Mac with does not expose internal docker IP. To fix it you will need following the steps in <<Solving bug in Mac OS to use Docker-Compose>>
+
+==== Solving bug in Mac OS to use Docker-Compose
+
+. Expose the Keycloak port in the link:./docker-compose/docker-compose-v2.1.yaml[docker-compose-v2.1]
++
```

Comment:
@danielpassos this step is not 100% clear for me. How to  Expose the Keycloak port? Is not it done by default already? What exactly we need to do here? Could you help by describing it?

Reply:
It's not needed anymore. I have sent a PR [exposing the port](https://github.com/aerogear/aerogear-unifiedpush-server/blob/master/docker-compose/docker-compose.yaml#L51) already.

Reply:
Actually I think we should avoid the docker-compose details here and link this section to the docker-compose folder README